primitives/io/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sp-io"
-version = "2.0.1"
+version = "2.0.0"
```

Comment:
Why downgrade?

Reply:
I had concerns about the version but both versions work.
Put 2.0.1 back as it's safer to keep the existing version.

Reply:
RIght. No need to downgrade without a specific reason.